2d_classification/mednist_tutorial.ipynb

```diff
@@ -138,8 +125,8 @@
    "source": [
     "## Setup data directory\n",
     "\n",
-    "You can specify a directory with the `MONAI_DATA_DIRECTORY` environment variable.  \n",
```

Comment:
All these blanks in Markdown doc are to make the next sentence in a new line, that's Markdown gramma I think.
Thanks.

Reply:
This is going to be tricky to solve, but thanks for pointing it out.

Reply:
I've committed a temporary fix f7deb35 pending answer from here: https://github.com/mwouts/jupytext/issues/723.

Reply:
OK, could you please update the PR to ignore the blanks?

Reply:
I've done that, but I'll have to revert all these accidental changes manually.